fs/src/io_uring/sequential_file_reader.rs

```diff
@@ -86,11 +86,11 @@ impl<'sp> SequentialFileReaderBuilder<'sp> {
     ///
     /// Initially the reader is idle and starts reading after the first file is added.
     /// The reader will execute multiple `read_capacity` sized reads in parallel to fill the buffer.
-    pub fn build_with_buffer<B: AsMut<[u8]>>(
+    pub fn build_with_buffer(
```

Comment:
I think after this change it's quite unlikely that reader will be built using provided buffer, so I would make this function private for now (or `pub(crate)` if it's used anywhere else e.g. for tests)

Reply:
https://github.com/anza-xyz/agave/pull/9531/commits/db2b84893f162a1904a07f3c7fdc45a8ca12d231